core/metadata-3.2.md

```diff
@@ -120,6 +132,23 @@ using `ERR_KEYNOPERMISSION` with an asterisk (`*`) in the `<Key>` field.
 
 *Errors*: `ERR_KEYNOPERMISSION`
 
+### METADATA SYNC
+
+This subcommand requests the full synchronization of metadata associated with
+the given target. It is only to be used when `metadata-notify` is enabled
+by the client.
+
```

Comment:
That said, LIST doesn't let you get the metadata for an entire channel member list in one go.